### ISSUE TITLE
es5 string#split type

### ISSUE THREAD

**Issue by tanhh326** [microsoft/TypeScript#55568]
### 🔎 Search Terms

[mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)

### 🕗 Version & Regression Information

current

### 💻 Code

https://github.com/microsoft/TypeScript/blob/08e0eb74e6eb1acac26221363c1b1fffe5102d47/src/lib/es5.d.ts#L473
```ts
   "x,y,z".split();
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split

### 🙁 Actual behavior

The `separator` here should be omitted, right?

### 🙂 Expected behavior

typescript does not report an error

**Comment by MartinJohns** [comment]
Duplicate of #38331.
